src/constants.tsx

```diff
@@ -41,6 +41,7 @@ export const BLOOD_SCREENING_ACTIVITY_CODE = 'bloodScreening';
 export const BEDNET_DISTRIBUTION_ACTIVITY_CODE = 'bednetDistribution';
 export const LARVAL_DIPPING_ACTIVITY_CODE = 'larvalDipping';
 export const MOSQUITO_COLLECTION_ACTIVITY_CODE = 'mosquitoCollection';
+export const TEST = 'test';
```

Comment:
Do we need to rename this to somehow indicate it is used for GA, in case we think there's a danger it may be inadvertently changed/translated etc?